play.pokemonshowdown.com/src/panel-teambuilder-team.tsx

```diff
@@ -239,6 +256,23 @@ class TeamPanel extends PSRoomPanel<TeamRoom> {
 					/>
 				</label>
 				<TeamTextbox team={team} />
+				{info && (info.resources.length || info.url) && (
```

Comment:
```suggestion
				{!!(info && (info.resources.length || info.url)) && (
```

The `&&` pattern only works with bools. You could accidentally insert a `0` into your output if you're not careful.

Reply:
Right, right. I always forget that sometimes that returns numbers.